### PR TITLE
add matplotlib, cairo & PyCairo dependencies for graph-tool 2.26 + enhance sanity check

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

follow-up for #5779

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 3 out of 4 (4 easyconfigs in this PR)
node2553.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/685de1fa885d3318f4dfbc78102c8c40 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 3 out of 4 (4 easyconfigs in this PR)
node2010.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/9fe016b884db37e2f3b952675f0c4e99 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2553.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/c2743f73df56f039ac0ffd0569e4b6fb for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2010.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/24057a5f19438b7092f4c20bf9e3639a for a full test report.

File: easybuild/easyconfigs/l/libsigc++/libsigc++-2.10.0-GCCcore-6.4.0.eb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+homepage = 'http://www.gtk.org/'
+description = """The libsigc++ package implements a typesafe callback system for standard C++."""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://ftp.gnome.org/pub/gnome/sources/%(name)s/%(version_major_minor)s/']
+sources = ['%(namelower)s-%(version)s.tar.xz']
+
+builddependencies = [('binutils', '2.28')]
```
Comment (vanzod) [inline easybuild/easyconfigs/l/libsigc++/libsigc++-2.10.0-GCCcore-6.4.0.eb]: `M4-1.4.18` is also needed as build dependency

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2553.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/0203884d9675f213ead5026e11fa87b0 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2010.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/e607d4676991234c9292de0fbc7c75b2 for a full test report.

File: easybuild/easyconfigs/c/cairomm/cairomm-1.12.2-GCCcore-6.4.0.eb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+version = '1.12.2'
+
+homepage = 'http://cairographics.org'
+description = "The Cairomm package provides a C++ interface to Cairo."
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+
+source_urls = ['http://cairographics.org/releases/']
+sources = [SOURCE_TAR_GZ]
+
```
Comment (vanzod) [inline easybuild/easyconfigs/c/cairomm/cairomm-1.12.2-GCCcore-6.4.0.eb]: Missing checksum

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2436.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/9ad49d1d6d41080554845b622615fe76 for a full test report.

**Pull request merged**